app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,37 @@
+<% if page.main_properties_present? %>
+  <div class="crayons-card p-5">
+    <% if page.image %>
+      <div class="s:max-w-50 mb-4">
+        <img alt="" src="<%= page.image %>" height="auto" loading="lazy">
+      </div>
+    <% end %>
+    <div class="text-styles">
+      <h2 class="fs-xl fw-bold lh-tight mb-2">
+        <a href="<%= page.url %>" class="ltag" target="_blank" rel="noopener noreferrer">
+          <%= page.title %>
+        </a>
+      </h2>
+    </div>
+    <div>
+      <% if page.description %>
+        <p class="truncate-at-2 mb-2">
+          <%= page.description %>
+        </p>
+      <% end %>
+    </div>
+    <div class="color-secondary fs-s flex items-center">
+      <% if page.images.favicon %>
```

Comment:
I decided to remove `site_name` from the embed. Some site names are terribly long, which skews the layout on smaller screens (and which I do not think we should perform CSS acrobatics to overcome 🤷🏽‍♀️)

I think the embed has sufficient information. @ludwiczakpawel